dissect/target/plugins/os/windows/regf/mru.py

```diff
@@ -321,7 +308,9 @@ def msoffice(self) -> Iterator[MSOfficeMRURecord]:
                         pass
 
 
-def parse_mru_key(target, key, record):
+def parse_mru_key(
+    target: Target, key: RegistryKey, record: UserRegistryRecordDescriptor
```

Comment:
```suggestion
    target: Target, key: RegistryKey, record: TargetRecordDescriptor
```
Technically not very correct still but it's the best we can do with the current typing on `flow.record` stuff.